examples/basics/visuals/polygon_prototype.py

```diff
@@ -0,0 +1,49 @@
+from vispy import app
```

Comment:
So, what do I do with this? I was testing out the rewritten polygons in this file.